src/Helpers/EmailHelpers/EmailHelper.cs

```diff
@@ -0,0 +1,45 @@
+﻿using form_builder.Configuration;
+using form_builder.Constants;
+using form_builder.Helpers.Session;
+using form_builder.Providers.Transforms.EmailConfiguration;
+using form_builder.Services.MappingService;
+using form_builder.Services.MappingService.Entities;
+using Newtonsoft.Json;
+using StockportGovUK.NetStandard.Gateways;
```

Comment:
remove unused "using" statements... could do a clean up perhaps, see what else could be removed while you're at it.